lib/bindings/http/binary_0_2.js

```diff
@@ -2,7 +2,7 @@ var axios = require("axios");
 var empty = require("is-empty");
 
 function HTTPBinary(configuration){
-  this.config = configuration;
+  this.config = JSON.parse(JSON.stringify(configuration));
```

Comment:
Check out the [line #15](https://github.com/cloudevents/sdk-javascript/blob/77bad8d07f3c707446a5414d0a1e658c2752f8cf/lib/bindings/http/binary_0_2.js#L15). The same way.